Library/Formula/go.rb

```diff
@@ -76,9 +93,10 @@ def caveats; <<-EOS.undent
     `go vet` and `go doc` are now part of the go.tools sub repo:
       http://golang.org/doc/go1.2#go_tools_godoc
 
-    To get `go vet` and `go doc` run:
+    `godoc` has been installed for you unless you requested otherwise.
+
+    To get `go vet` run:
       go get golang.org/x/tools/cmd/vet
-      go get golang.org/x/tools/cmd/godoc
```

Comment:
Maybe we should install `go vet` too? Also, `unless you requested otherwise.`: we know if they did by `build.with?`.